2_process/src/process_data.R

```diff
@@ -0,0 +1,25 @@
+############## Process data  ######################
+
+project_output_dir <- '1_process/out'
```

Comment:
You should delete this line. You have a similar line in README.R that is doing the same thing but in a better place. This line gets executed when you run `source('2_process/src/process_data.R')`, but then the value of `project_output_dir` gets overwritten almost immediately (before getting used) in the next lines of README.R

Reply:
outdated